system/jlib/jtrace.cpp

```diff
@@ -1242,6 +1339,68 @@ class CChildSpan : public CSpan
 
 protected:
     CSpan * localParentSpan = nullptr;
+    //int64_t thresholdDurationMS = 0;
+    //SpanTimeStamp rawStartTimeStamp;
+    //std::chrono::system_clock::time_point startTP;
+    //TernaryBool thresholdMet = TernaryBool::Uninitialized;
+};
+
+class CConditionalChildSpan : public CChildSpan
+{
```

Comment:
conditionalchildspan doesn't actually initialize a span obj until the end after we can can conclude if the threshold has been met, then we immediately end the span